### PR TITLE
🌐 Initialize translations for Traditional Chinese

### PR DESCRIPTION
**Pull request by hsuanchi**

### CODE REVIEW

**Comment by hsuanchi** [comment]
Hello, @tiangolo 

We want to create the language for zh-hant (Mandarin), but the language will be limited to 2 letters. What are your recommended approaches for handling languages that have more than two letters?

**Comment by tiangolo** [comment]
Thanks @hsuanchi! Up to now, I've avoided language localizations as there aren't yet enough translations of the first language, for example, for `pt-PT`.

The first question would be, how different is it from `zh`? Would `zh` not be understandable by people who speak `zh-hant`?

The second question is, are there others (at least other two, in total with you at least three) that are willing to help with those translations to `zh-hant`? Because I need to get 2 approvals to merge, and if there are now two variants of `zh`, it's probably gonna be even more difficult to get approvals for both.

There are currently 63 PRs for `zh` awaiting for reviewers: https://github.com/tiangolo/fastapi/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Alang-zh

Let me know what you think!

**Comment by hsuanchi** [comment]
Hello @tiangolo ,

Thank you for your swift reply. To address your first question, zh is generally considered to be Simplified Chinese (zh-hans), while zh-hant refers to Traditional Chinese. Although the two variants share similarities, they differ significantly in terms of characters and some terminology. People from regions such as Taiwan, Hong Kong, and Macau typically use zh-hant and may find zh less intuitive to read.

As for your second question, we have a robust team of over 10 developers who use FastAPI on a daily basis. Some of our members have also contributed to the Traditional Chinese translation of Python documentation ([python-docs-zh-tw](https://github.com/python/python-docs-zh-tw)). We're confident that we'll be able to efficiently review and contribute to the zh-hant localization for FastAPI.

Thank you once again, and looking forward to your reply.

**Comment by SonnyYou** [comment]
Hi @tiangolo ,

I would like to kindly inform you that I am part of the same team as @hsuanchi , and we are wholeheartedly dedicated to supporting the translation of the content into `zh-hant`. Let's collaborate to make it even more exceptional!

**Comment by pths411027** [comment]
I'm an avid user of FastAPI and would be thrilled to review @hsuanchi  's translation. Additionally, I'd be honored to contribute to the translation efforts for FastAPI, aiming to make it even more accessible and developer-friendly for the Chinese-speaking community."

**Comment by winonachiang** [comment]
Fastapi basically consists of my day-to-day life and I'm confident that I can dedicate myself to contributing to the translations to zh-hant as zh-hant is my first language.
It'd be my pleasure to review @hsuanchi's translation and make the development of Fastapi more friendly to the Chinese-speaking community.

**Comment by amber14liu** [comment]
Hello @tiangolo,

I am in the same team as @hsuanchi. I am very eager to assist in the zh-hant localization to make FastAPI more accessible for the Traditional Chinese community. Looking forward to hearing from you!

**Comment by jsbao0428** [comment]
I am a user of FastAPI from Taiwan, and I am very eager to promote FastAPI among Traditional Chinese-speaking users. This way, more users can read the documentation in their native language, which would lower the barrier to entry.

**Comment by DavidChen-99** [comment]
Hi @tiangolo,

If I could contribute to the translation efforts of FastAPI, making community documents more clear and intuitive for developers using the zh-hant language, I find it highly meaningful.

**Comment by enzochang17** [comment]
Dear @tiangolo,

I am extremely excited and looking forward to seeing FastAPI in Traditional Chinese.  The purpose of this initiative is to assist developers in Taiwan in using and understanding FastAPI more easily. We are committed to continuing to support and contribute to FastAPI.

**Comment by enzoapu** [comment]
Dear @tiangolo,

I am extremely excited and looking forward to seeing FastAPI in Traditional Chinese. The purpose of this initiative is to assist developers in Taiwan in using and understanding FastAPI more easily. We are committed to continuing to support and contribute to FastAPI.

**Comment by isaackwok** [comment]
As a software engineer from Hong Kong and Taiwan, I can tell that FastAPI is important for Traditional Chinese software community, we have a large community here working on Python with FastAPI in Taiwan.

Can't wait to see more Traditional Chinese resources and contributions so FastAPI can be widely promoted.

**Comment by mattwang44** [comment]
Traditional Chinese (`zh_hant`) and Simplified Chinese (`zh_hans`) are different *writing systems* of Chinese. Both are categorized under the `zh` code in ISO 639-1, and using a two-character code alone is insufficient to differentiate between the two. (Similarly, I'd say the term "Mandarin" in the PR title is not accurate enough.) They have different histories and cultures. Both are official in their regions and are widely used with rich literature and media. Their users generally have a strong preference for one writing system over the other based on their background. Though Mandarin is mostly spoken in both regions that use Traditional and Simplified Chinese characters, regional variations exist between both systems.

In the field of software technical document translation, many prominent resources offer both Traditional and Simplified Chinese versions simultaneously. Examples include [Python docs](https://docs.python.org/zh-tw/3/), [MDN docs](https://developer.mozilla.org/zh-TW/), [Microsoft docs](https://learn.microsoft.com/zh-tw/docs/), [AWS docs](https://docs.aws.amazon.com/zh_tw/), [React docs](https://zh-hant.legacy.reactjs.org/), [Angular docs](https://angular.tw/), and more. Although the momentum for zh_hant translation might not be as extensive as zh_hans, it's undeniable that Traditional Chinese is a legitimate and important language for these translations.

As a Python backend engineer and a member of Python document translation community, I'm  more than willing to help with translating the FastAPI doc or reviewing for the translation PRs. (Actually, I [tried once before](https://github.com/tiangolo/fastapi/discussions/9758) lol).

**Comment by kyle0828** [comment]
Hi @tiangolo , 

I am a software engineer who works with @hsuanchi. I understand your concern about the use of Traditional Chinese (`zh-Hant`) for translations. Here's a more detailed explanation of why it's essential to consider using `zh-Hant` for FastAPI translations:

`zh-Hant`, which represents Traditional Chinese, is crucial for ensuring the accessibility and readability of FastAPI content for a specific audience. Traditional Chinese characters are primarily used in regions like Taiwan, Hong Kong, and among overseas Chinese communities. If your project has users in these areas or if your goal is to be inclusive of these regions, providing translations in `zh-Hant` is a necessity.

Here's why `zh-Hant` matters:
1. **Cultural Sensitivity**: Using Traditional Chinese characters is a matter of cultural sensitivity and respect. Some users in regions where Traditional Chinese is the norm may find it more comfortable and culturally appropriate when content is presented in `zh-Hant`.

2. **Audience Reach**: Including `zh-Hant` extends your reach and ensures that FastAPI is accessible to a broader audience. It allows you to engage users who may prefer or are more familiar with Traditional Chinese characters.

3. **User Experience**: Users tend to have a better user experience when content is in their preferred language variant. It enhances user satisfaction and encourages adoption.

In summary, providing translations in `zh-Hant` alongside `zh` is a strategic move to make FastAPI more inclusive, culturally sensitive, and user-friendly for a wider audience. It demonstrates your commitment to engaging with users from different regions and respecting their language and cultural preferences.

We hope you consider the importance of using `zh-Hant` as part of the translation efforts for FastAPI. Thank you for your understanding and support.

**Comment by hsuanchi** [comment]
Hi @tiangolo,  are there any updates? Are we ready to begin work on the Traditional Chinese version?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 84b953ba9a18cb60f296ae9bdbdada60937e7562 at: https://3397a9ff.fastapitiangolo.pages.dev

**Review by mattwang44 — COMMENTED** [review]
Here is the initial review of the translation part.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 953ad1489ace9cae73f2fdcf3dfa1df4e94aec20 at: https://b5531961.fastapitiangolo.pages.dev

**Comment by hsuanchi** [comment]
> Here is the initial review of the translation part.

Thanks for helping with the review 🫡🫡🫡, all amendments are now complete.

File: docs/zh-hant/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+你使用 Python 標準類型來完成聲明。
+
+你不需要學習新的語法、類別、方法或函式庫等等。
+
+只需要使用 **Python 3.8 以上的版本**。
+
+舉個範例，比如宣告 int 的型別：
+
+```Python
+item_id： int
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:339]: code 裡面的冒號不小心被改到了

File: docs/zh-hant/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+
+在進行一次宣告後，你將獲得：
+
+- 編輯器支援：
+  - 自動補全
+  - 型別檢查
+- 資料驗證：
+  - 驗證失敗時自動生成清楚的錯誤訊息
+  - 可驗證多層巢狀的 JSON 物件
+- <abbr title="也被稱為： 序列化或解析">轉換</abbr> 輸入的資料： 轉換來自網路請求到 Python 數據類型。包含以下數據：
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:356]: 這個沒改到，是漏掉了嗎？
https://github.com/tiangolo/fastapi/pull/10505#discussion_r1390608394

File: docs/zh-hant/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+
+使用瀏覽器開啟 <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>。
+
+- ReDoc API 文件會自動更新，並加入新的參數和 body 請求：
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-06-redoc-02.png)
+
+### 總結
+
+總結來說， 你就像宣告函式的參數類型一樣，只宣告了一次請求參數和請求主體參數等類型。
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:328]: 還有不少地方的 type 被翻成「類型」而非「型別」，要再麻煩檢查一下
```suggestion
總結來說， 你就像宣告函式的參數型別一樣，只宣告了一次請求參數和請求主體參數等型別。
```

File: docs/zh-hant/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+
+"_如果您想學習一種用於構建 REST API 的**現代框架**，不能錯過 **FastAPI** [...] 它非常快速、且易於使用和學習 [...]_"
+
+"_我們的 **APIs** 已經改用 **FastAPI** [...] 我想你會喜歡它 [...]_"
+
+<div style="text-align: right; margin-right: 10%;">Ines Montani - Matthew Honnibal - <strong><a href="https://explosion.ai" target="_blank">Explosion AI</a> founders - <a href="https://spacy.io" target="_blank">spaCy</a> creators</strong> <a href="https://twitter.com/_inesmontani/status/1144173225322143744" target="_blank"><small>(ref)</small></a> - <a href="https://twitter.com/honnibal/status/1144031421859655680" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_如果有人想要建立一個生產環境的 Python API，我強烈推薦 **FastAPI**. 它**設計精美**，**使用簡單**且**高度可擴充**，它已成為我們 API 優先開發策略中的**關鍵組件**，並且驅動了許多自動化服務，例如我們的 Virtual TAC Engineer。_"
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:102]:
```suggestion
"_如果有人想要建立一個生產環境的 Python API，我強烈推薦 **FastAPI**，它**設計精美**，**使用簡單**且**高度可擴充**，它已成為我們 API 優先開發策略中的**關鍵組件**，並且驅動了許多自動化服務，例如我們的 Virtual TAC Engineer。_"
```

File: docs/zh-hant/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,470 @@
...
+
+<div style="text-align: right; margin-right: 10%;">Timothy Crosley - <strong><a href="https://www.hug.rest/" target="_blank">Hug</a> creator</strong> <a href="https://news.ycombinator.com/item?id=19455465" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_如果您想學習一種用於構建 REST API 的**現代框架**，不能錯過 **FastAPI** [...] 它非常快速、且易於使用和學習 [...]_"
+
+"_我們的 **APIs** 已經改用 **FastAPI** [...] 我想你會喜歡它 [...]_"
+
+<div style="text-align: right; margin-right: 10%;">Ines Montani - Matthew Honnibal - <strong><a href="https://explosion.ai" target="_blank">Explosion AI</a> founders - <a href="https://spacy.io" target="_blank">spaCy</a> creators</strong> <a href="https://twitter.com/_inesmontani/status/1144173225322143744" target="_blank"><small>(ref)</small></a> - <a href="https://twitter.com/honnibal/status/1144031421859655680" target="_blank"><small>(ref)</small></a></div>
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:98]:
```suggestion
<div style="text-align: right; margin-right: 10%;">Ines Montani - Matthew Honnibal - <strong><a href="https://explosion.ai" target="_blank">Explosion AI</a> 創辦人 - <a href="https://spacy.io" target="_blank">spaCy</a> creators</strong> <a href="https://twitter.com/_inesmontani/status/1144173225322143744" target="_blank"><small>(ref)</small></a> - <a href="https://twitter.com/honnibal/status/1144031421859655680" target="_blank"><small>(ref)</small></a></div>
```

File: docs/zh-hant/docs/index.md [hunk 6]
```diff
@@ -0,0 +1,470 @@
...
+---
+
+FastAPI 是一個現代、快速（高效能）的 web 框架，用於 Python 3.8+ 並採用標準 Python 類別提示。
+
+主要特點包含：
+
+- **快速**： 非常高的效能，可與 **NodeJS** 和 **Go** 效能相當 (歸功於 Starlette and Pydantic)。 [FastAPI 是最快的 Python web 框架之一](#performance)。
+- **快速開發**： 提高開發功能的速度約 200% 至 300%。 \*
+- **更少的 Bug**： 減少約 40% 的人為（開發者）導致的錯誤。 \*
+- **直觀**： 具有出色的編輯器支援，處處都有<abbr title="也被稱為自動完成、IntellisSense">自動補全</abbr>以減少偵錯時間。
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:37]:
```suggestion
- **直觀**： 具有出色的編輯器支援，處處都有<abbr title="也被稱為自動完成、IntelliSense">自動補全</abbr>以減少偵錯時間。
```

File: docs/zh-hant/docs/index.md [hunk 7]
```diff
@@ -0,0 +1,470 @@
...
+
+---
+
+**文件**： <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**程式碼**： <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI 是一個現代、快速（高效能）的 web 框架，用於 Python 3.8+ 並採用標準 Python 類別提示。
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:30]:
```suggestion
FastAPI 是一個現代、快速（高效能）的 web 框架，用於 Python 3.8+ 並採用標準 Python 型別提示。
```

File: docs/zh-hant/docs/index.md [hunk 8]
```diff
@@ -0,0 +1,470 @@
...
+在進行一次宣告後，你將獲得：
+
+- 編輯器支援：
+  - 自動補全
+  - 型別檢查
+- 資料驗證：
+  - 驗證失敗時自動生成清楚的錯誤訊息
+  - 可驗證多層巢狀的 JSON 物件
+- <abbr title="也被稱為： 序列化或解析">轉換</abbr> 輸入的資料： 轉換來自網路請求到 Python 數據類型。包含以下數據：
+  - JSON.
```
Comment (mattwang44) [inline docs/zh-hant/docs/index.md:357]:
```suggestion
  - JSON
```

**Review by mattwang44 — COMMENTED** [review]
translation review part 2

File: docs/zh-hant/docs/index.md (hunk 2 above)
Comment (hsuanchi) [reply docs/zh-hant/docs/index.md]: 這個場景覺得還是維持在原本的「解析」，會比「剖析」來得更精確一些

**Comment by tiangolo** [comment]
📝 Docs preview for commit 73e95a72238dfc79c8d895ced60692e6fd8839a9 at: https://d5229374.fastapitiangolo.pages.dev

File: docs/zh-hant/docs/index.md [hunk 9]
```diff
@@ -0,0 +1,470 @@
...
+**程式碼**： <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI 是一個現代、快速（高效能）的 web 框架，用於 Python 3.8+ 並採用標準 Python 型別提示。
+
+主要特點包含：
+
+- **快速**： 非常高的效能，可與 **NodeJS** 和 **Go** 效能相當 (歸功於 Starlette and Pydantic)。 [FastAPI 是最快的 Python web 框架之一](#performance)。
+- **快速開發**： 提高開發功能的速度約 200% 至 300%。 \*
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:35]: 我覺得這裡使用"極速開發"更適合，也能與前一點的"快速"區分開來

File: docs/zh-hant/docs/index.md [hunk 10]
```diff
@@ -0,0 +1,470 @@
...
+- **更少的 Bug**： 減少約 40% 的人為（開發者）導致的錯誤。 \*
+- **直觀**： 具有出色的編輯器支援，處處都有<abbr title="也被稱為自動完成、IntelliSense">自動補全</abbr>以減少偵錯時間。
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:37]: "直覺"看起來更適合

File: docs/zh-hant/docs/index.md [hunk 11]
```diff
@@ -0,0 +1,470 @@
...
+{% endfor %}
+{% endif %}
+
+<!-- /sponsors -->
+
+<a href="https://fastapi.tiangolo.com/fastapi-people/#sponsors" class="external-link" target="_blank">其他贊助商</a>
+
+## 評價
+
+"_[...] 近期大量的使用 **FastAPI**。 [...] 目前正在計畫在微軟團隊的機器學習服務中導入。其中一些正在整合到核心的 **Windows** 產品和一些 **Office** 產品。_"
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:64]: 原文中有將`**微軟**`和`**機器學習**`設為粗體

File: docs/zh-hant/docs/index.md (hunk 9 above)
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:35]:
```suggestion
- **極速開發**： 提高開發功能的速度約 200% 至 300%。 \*
```

File: docs/zh-hant/docs/index.md (hunk 10 above)
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:37]:
```suggestion
- **直覺**： 具有出色的編輯器支援，處處都有<abbr title="也被稱為自動完成、IntelliSense">自動補全</abbr>以減少偵錯時間。
```

File: docs/zh-hant/docs/index.md (hunk 11 above)
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:64]:
```suggestion
"_[...] 近期大量的使用 **FastAPI**。 [...] 目前正在計畫在**微軟**團隊的**機器學習**服務中導入。其中一些正在整合到核心的 **Windows** 產品和一些 **Office** 產品。_"
```

Comment (SonnyYou) [reply docs/zh-hant/docs/index.md:64]: 原文中有將`**微軟**`和`**機器學習**`設為粗體

**Review by SonnyYou — COMMENTED** [review]
Here are some suggestions

File: docs/zh-hant/docs/index.md [hunk 12]
```diff
@@ -0,0 +1,470 @@
...
+- 驗證 `GET` 和 `PUT` 請求中的 `item_id` 是否是 `int` 型別。
+  - 如果驗證失敗，將會返回清楚有用的錯誤訊息。
+- 查看 `GET` 請求中是否有命名為 `q` 的查詢參數 (例如 `http://127.0.0.1:8000/items/foo?q=somequery`)。
+  - 因為 `q` 參數被宣告為 `= None`，所以是選填的。
+  - 如果沒有宣告 `None`，則此參數將會是必填 (例如 `PUT` 範例的請求 body)。
+- 對於 `PUT` 的請求 `/items/{item_id}`，將會讀取 body 為 JSON：
+  - 驗證是否有必填屬性 `name` 且型別是 `str`。
+  - 驗證是否有必填屬性 `price` 且型別是 `float`。
+  - 驗證是否有選填屬性 `is_offer` 且型別是 `bool`。
+  - 以上驗證都適用於多層次巢狀 JSON 物件。
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:388]:
```suggestion
    - 驗證是否有必填屬性 `name` 且型別是 `str`。
    - 驗證是否有必填屬性 `price` 且型別是 `float`。
    - 驗證是否有選填屬性 `is_offer`，如果有的話型別是 `bool`。
    - 以上驗證都適用於多層次巢狀 JSON 物件。
```

File: docs/zh-hant/docs/index.md [hunk 13]
```diff
@@ -0,0 +1,470 @@
...
+- 自動轉換 JSON 格式。
+- 透過 OpenAPI 文件來記錄所有內容，可以被用於：
+  - 互動式文件系統。
+  - 自動為多種程式語言生成用戶端的程式碼
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:392]:
```suggestion
    - 互動式文件系統。
    - 自動為多種程式語言生成用戶端的程式碼。
```

File: docs/zh-hant/docs/index.md [hunk 14]
```diff
@@ -0,0 +1,470 @@
...
+- 簡單且非常容易使用的 **<abbr title="也被稱為元件、資源、提供者、服務或是注入">依賴注入</abbr>** 系統。
+- 安全性和身份驗證，包含提供支援 **OAuth2**、**JWT tokens** 和 **HTTP Basic** 驗證。
+- 更進階 (但同樣簡單) 的宣告 **多層次的巢狀 JSON 格式** (感謝 Pydantic)。
+- **GraphQL** 與 <a href="https://strawberry.rocks" class="external-link" target="_blank">Strawberry</a> 以及其他的相關函式庫進行整合。
+- 更多其他的功能 (感謝 Starlette) 像是：
+  - **WebSockets**
+  - 於 HTTPX 和 `pytest` 的非常簡單測試
+  - **CORS**
+  - **Cookie Sessions**
+  - ...以及更多
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:436]:
```suggestion
    - **WebSockets**
    - 於 HTTPX 和 `pytest` 的非常簡單測試
    - **CORS**
    - **Cookie Sessions**
    - ...以及更多
```

File: docs/zh-hant/docs/index.md [hunk 15]
```diff
@@ -0,0 +1,470 @@
...
+- 對於 `PUT` 的請求 `/items/{item_id}`，將會讀取 body 為 JSON：
+  - 驗證是否有必填屬性 `name` 且型別是 `str`。
+  - 驗證是否有必填屬性 `price` 且型別是 `float`。
+  - 驗證是否有選填屬性 `is_offer` 且型別是 `bool`。
+  - 以上驗證都適用於多層次巢狀 JSON 物件。
+- 自動轉換 JSON 格式。
+- 透過 OpenAPI 文件來記錄所有內容，可以被用於：
+  - 互動式文件系統。
+  - 自動為多種程式語言生成用戶端的程式碼
+- 提供兩種交互式文件介面
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:393]:
```suggestion
- 提供兩種交互式文件介面。
```

File: docs/zh-hant/docs/index.md [hunk 16]
```diff
@@ -0,0 +1,470 @@
...
+
+---
+
+雖然我們只敘述了表面的功能，但其實你已經理解了它是如何運行。
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:397]:
```suggestion
雖然我們只敘述了表面的功能，但其實你已經理解了它是如何執行。
```

File: docs/zh-hant/docs/index.md [hunk 17]
```diff
@@ -0,0 +1,470 @@
...
+- 更多其他的功能 (感謝 Starlette) 像是：
+  - **WebSockets**
+  - 於 HTTPX 和 `pytest` 的非常簡單測試
+  - **CORS**
+  - **Cookie Sessions**
+  - ...以及更多
+
+## 效能
+
+來自獨立機構 TechEmpower 的測試結果，顯示在 Uvicorn 運行下的 **FastAPI** 是 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">最快的 Python 框架之一</a>， 僅次於 Starlette 和 Uvicorn 本身 (兩者是 FastAPI 的底層)。 (\*)
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:440]:
```suggestion
來自獨立機構 TechEmpower 的測試結果，顯示在 Uvicorn 執行下的 **FastAPI** 是 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">最快的 Python 框架之一</a>， 僅次於 Starlette 和 Uvicorn 本身 (兩者是 FastAPI 的底層)。 (\*)
```

File: docs/zh-hant/docs/index.md [hunk 18]
```diff
@@ -0,0 +1,470 @@
...
+
+想了解更多訊息，可以參考 <a href="https://fastapi.tiangolo.com/benchmarks/" class="internal-link" target="_blank">測試結果</a>。
+
+## 可選的依賴套件
+
+用於 Pydantic：
+
+- <a href="https://github.com/JoshData/python-email-validator" target="_blank"><code>email_validator</code></a> - 用於電子郵件驗證。
+- <a href="https://docs.pydantic.dev/latest/usage/pydantic_settings/" target="_blank"><code>pydantic-settings</code></a> - 用於設置管理。
```
Comment (SonnyYou) [inline docs/zh-hant/docs/index.md:449]:
```suggestion
- <a href="https://docs.pydantic.dev/latest/usage/pydantic_settings/" target="_blank"><code>pydantic-settings</code></a> - 用於設定管理。
```

File: docs/zh-hant/docs/index.md [hunk 19]
```diff
@@ -0,0 +1,470 @@
...
+
+---
+
+## **Typer**，命令列中的 FastAPI
+
+<a href="https://typer.tiangolo.com" target="_blank"><img src="https://typer.tiangolo.com/img/logo-margin/logo-margin-vector.svg" style="width: 20%;"></a>
+
+如果你不是在開發網頁 API，而是正在開發一個在終端機中運行的<abbr title="Command Line Interface">命令列</abbr>應用程式，不妨嘗試 <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>。
+
+**Typer** 是 FastAPI 的小兄弟。他立志成為命令列的 **FastAPI**。 ⌨️ 🚀
```
[the remark this answers is not in the document]
Comment (hsuanchi) [reply docs/zh-hant/docs/index.md:114]: 小兄弟還是比較貼切 🥹🥹

**Comment by tiangolo** [comment]
📝 Docs preview for commit 66cba69728d587ee07cbb3b08168e0c252f9e50f at: https://c9489c35.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Amazing! If you all work together you could have all the FastAPI docs translated to Traditional Chinese in no time! 🎉 

I updated the script to handle docs to take this into account, and I added this discussion: https://github.com/tiangolo/fastapi/discussions/10949

When there's a new PR for Traditional Chinese and I add the label, it will be automatically posted there. You could subscribe to that discussion to get notified when there's a new PR to review. 🤓 

Thank you all for your help!

Once this PR is ready and has two approving reviews I'll merge it. The same with the next future PRs. 🚀

**Comment by tiangolo** [comment]
📝 Docs preview for commit efc2197589ec94dda13015d4a17d80d9bc4b6334 at: https://be6f65a0.fastapitiangolo.pages.dev

**Review by mattwang44 — APPROVED** [review]
LGTM

**Review by SonnyYou — APPROVED** [review]
LGTM

**Comment by tiangolo** [comment]
Awesome, thank you @hsuanchi ! 🍰 

And thanks for the reviews @SonnyYou, @mattwang44 ☕ 🍪
